routes/threadsRouter.js

```diff
@@ -1,59 +1,59 @@
 const express = require('express');
 const router = express.Router();
-
+const passport = require('passport');
 const bodyParser = require('body-parser');
-const {passport} = require('./userRouter')
-const {Threads} = require('../models/threadModel');
+// const { passport } = require('../server');
+const { Threads } = require('../models/threadModel');
 
 
-router.get('/', (req, res) => { 
+router.get('/', passport.authenticate('basic', { session: true }), (req, res) => {
```

Comment:
The `/threads` endpoint has been protected. After the first time login, we can guarantee that an unlogged in user should not be able to access threads.